src/askui/tools/android/tools.py

```diff
@@ -62,15 +63,44 @@ def __init__(self, agent_os_facade: AndroidAgentOsFacade) -> None:
                         "type": "integer",
                         "description": "The y coordinate of the tap gesture in pixels.",
                     },
+                    "number_of_taps": {
```

Comment:
please stick to the naming of the locate function in the Vision Agent

```suggestion
                    "repeat": {
```